data/scripts/molgenis-model-update/run.py

```diff
@@ -0,0 +1,93 @@
+from decouple import config
+from session import Session
+from update import TransformGeneral
+from update import TransformDataCatalogue
+from spaces import Spaces
+from zip_handling import Zip
+
+# Data model details
+DATA_MODEL_VERSION = config('MG_DATA_MODEL_VERSION')
+
+# Server details
+SERVER_URL = config('MG_SERVER_URL')
+SERVER_USERNAME = config('MG_SERVER_USERNAME')
+SERVER_PASSWORD = config('MG_SERVER_PASSWORD')
```

Comment:
I find it a risk to include username/password in a config. Can this be a commandline option? Or is that already how config works?

Reply:
the environment values are read from .env and .env is in .gitignore. but i can also rewrite to get the password from commandline.